roles/setup_base_config/templates/sourceme_site.sh.j2

```diff
@@ -14,8 +17,8 @@ alias deployed_env="cat {{ deployed_tool_versions }}"
 
 {% if site == "upps" %}
 export PATH={{ sw_path }}:$PATH
+module load bioinfo-tools bowtie2 fastq_screen
```

Comment:
in keeping with the `DDS-cli` pattern in the delivery service, would it be possible to only load these modules for the seqreports service through a wrapper? or is it required that the user also has these modules loaded in any way?

Reply:
Yes I think that should be possible, I'll look into that :+1:

Reply:
Note that seqreports uses a singularity container to run fastq_screen, so the module won't be used. 

If we only load the module to get the path to the references we could perhaps directly specify it ? `/sw/data/fastq_screen_data/20220330/miarka/FastQ_Screen_Genomes`